src/kernels/MIOpenLRNBwd.cl

```diff
@@ -161,6 +161,7 @@ MIOpenLRNWithinChannelBwd(const __global _FLOAT* top,
         }
     }
 
+    barrier(CLK_LOCAL_MEM_FENCE);
```

Comment:
@muralinr May I ask why it fixes the issue? Looks like a racing condition but where? :-(

P.S. a comment for Artem: Please add the config to the unit test too. Thanks! ;-P